### ISSUE TITLE
Can we convert the codebase into TypeScript?

### ISSUE THREAD

**Issue by AlexFreik** [naufaldi/next-landing-vpn#27]
The project looks very cool! My only issue is it is in JS...

**Comment by jaykeraliya0** [comment]
Yes, I agree with you. Also, if the owner permits, I'm willing to contribute.

**Comment by naufaldi** [comment]
Go for it! 
we can create separate branch for javascript and typescript. So, people can still new in this field can still learn in javascript or in typescript
